Src/Particle/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 target_compile_definitions(amrex PUBLIC $<BUILD_INTERFACE:AMREX_PARTICLES>)
 
-if (NOT ENABLE_DP_PARTICLES)
+if (AMReX_PARTICLES_PRECISION STREQUAL SINGLE)
```

Comment:
bug: "SINGLE"

Reply:
This has already been fixed in development